CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 **********
 
+[1.4.0] - 2022-10-21
+********************
+
+Changed
+=======
+* Remove override of auto.offset.reset on consumer (which will default to "latest"). New consumer groups will consume only messages that are sent after the group was initialized.
```

Comment:
I think this is heavy on the removal, which ultimately doesn't matter if we aren't even saying what was removed.  Why not something like:
```suggestion
* Updated auto.offset.reset on consumer to "latest" (the default value). New consumer groups will consume only messages that are sent after the group was initialized.
```